docs/portal-contribute.md

```diff
@@ -46,12 +46,34 @@ If you think something is wrong or something fundamental should change, discussi
 * [Issues in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/issues) 
 * [Discussions in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/discussions)
 
-### Add projects to showcase or builder tools
-Help to add good projects or builder tools. Please note the different guidelines for [adding showcases](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js) or [builder tools](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/builder-tools.js).
+### Add a project to showcase 
+Help to add good projects to the [showcase section](http://developers.cardano.org/showcase/). Please note the guidelines for adding new projects:
+* It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but nothing for this showcase section.
+* It has to run on Cardano mainnet.
+* It has to have a running product. (no presale, no protected pages, no coming soon messages)
+* It has to have enough community reputation.
+* It has to provide a unique value from existing showcase items. 
+* It has to have a stable domain name. (a random Netlify/Vercel domain is not allowed, no URL shortener, no app store links, or similar)
+* The GitHub account that adds the project must not be new. 
+* The GitHub account must have a history/or already be known in the Cardano community.
+* Describe what makes your project special, avoid phrases like "the first this and that". Granular details like which project was first is tribal attribute known to cause rift and conflicts.
+* Please refrain from adding NFT projects unless they are immensely different from what is already listed in terms of utility. We can't list hundrets of NFT. For details please follow or take part in the discussion [here](https://github.com/cardano-foundation/developer-portal/discussions/243). (TLDR: we are adding winners of the CNFT awards)
```

Comment:
```suggestion
* Please refrain from adding NFT projects unless they are immensely different from what is already listed in terms of utility. We can't list hundreds of NFT projects. For details please follow or take part in the discussion [here](https://github.com/cardano-foundation/developer-portal/discussions/243). (TL;DR: we are adding winners of the [CNFT Awards](https://www.cnft-awards.io/))
```
TL/DR is considered canonical (although TLDR is also acceptable): https://www.merriam-webster.com/dictionary/TL%3BDR